src/Database/Migrations/2020-12-28-223112_create_auth_tables.php

```diff
@@ -15,13 +15,18 @@ class CreateAuthTables extends Migration
      */
     private array $tables;
 
+    private array $attributes;
+    private bool $ifNotExists;
```

Comment:
Currently, we use `false`. I think we don't need to use a new variable, using `false` is enough.
e.g.
```php
$this->forge->createTable($this->tables['users'], false, $this->attributes);
```

Reply:
It is certainly preferable to have no property that is not needed.

Reply:
i have removed property